docs/msf/managing-store-configuration.mdx

```diff
@@ -143,15 +177,15 @@ If you wish to restore global defaults on a channel with a channel-specific conf
 * a comma-separated list of the property `keys` for the values you wish to reset.
 
 
-```http title="Example request: Reset channel-specific store profile settings to default"
+```http filename="Example request: Reset channel-specific store profile settings to default"
```

Comment:
```suggestion
```http filename="Example request: Reset channel-specific store profile settings to default" copy
```